src/server/init.rs

```diff
@@ -246,14 +248,24 @@ pub fn init(options: &Init) -> anyhow::Result<()> {
         match method.bootstrap(&settings) {
             Ok(()) => {}
             Err(e) => {
-                log::error!("Bootstrap error, cleaning up...");
-                method.clean_storage(&settings.storage)
-                    .map_err(|e| {
-                        log::error!("Cannot clean up storage {}: {}",
-                            settings.storage.display(), e);
-                    }).ok();
-                Err(e).context(format!("cannot bootstrap {}",
-                                       settings.storage.display()))?
+                if e.is::<CannotCreateService>() {
+                    eprintln!("Error: {:#}", e);
+                    eprintln!("Bootrapping complete, \
+                        but there was an error creating a service. \
+                        You can run server manually via: \n  \
+                        edgedb server start --foreground {}",
+                        settings.name.escape_default());
+                    return Err(ExitCode::new(2))?;
+                } else {
+                    log::error!("Bootstrap error, cleaning up...");
+                    method.clean_storage(&settings.storage)
+                        .map_err(|e| {
+                            log::error!("Cannot clean up storage {}: {}",
+                                settings.storage.display(), e);
+                        }).ok();
+                    Err(e).context(format!("cannot bootstrap {}",
```

Comment:
```suggestion
                    Err(e).context(format!("Cannot bootstrap {}",
```

Reply:
I think we always have something like `Error: cannot bootstrap...`. And while we have some capitalized errors, I think this is a mistake and we generally tend to lowercase them. Capitalized messages should be only used when we directly output with `log/println/eprintln`